packages/wonder-blocks-progress-spinner/src/components/__docs__/circular-spinner.stories.js

```diff
@@ -0,0 +1,181 @@
+// @flow
+import * as React from "react";
+import {StyleSheet, css} from "aphrodite";
+import type {StoryComponentType} from "@storybook/react";
+
+import Color from "@khanacademy/wonder-blocks-color";
+import {View} from "@khanacademy/wonder-blocks-core";
+import {CircularSpinner} from "@khanacademy/wonder-blocks-progress-spinner";
+import Spacing from "@khanacademy/wonder-blocks-spacing";
+import {Body, LabelLarge} from "@khanacademy/wonder-blocks-typography";
+
+import ComponentInfo from "../../../../../.storybook/components/component-info.js";
+import {name, version} from "../../../package.json";
+
+export default {
+    title: "ProgressSpinner/CircularSpinner",
+    component: CircularSpinner,
+    parameters: {
+        componentSubtitle: ((
+            <ComponentInfo name={name} version={version} />
+        ): any),
+        docs: {
+            description: {
+                component: null,
+            },
+            source: {
+                // See https://github.com/storybookjs/storybook/issues/12596
+                excludeDecorators: true,
+            },
+        },
+    },
+    decorators: [
+        (Story: any): React.Element<typeof View> => (
+            <View style={styles.example}>
+                <Story />
+            </View>
+        ),
+    ],
+};
+
+export const Default: StoryComponentType = (args) => (
+    <CircularSpinner {...args} />
+);
+
+export const Sizes: StoryComponentType = () => (
+    <table>
+        <tr>
+            <td>
+                <LabelLarge>xsmall</LabelLarge>
+            </td>
+            <td>
+                <LabelLarge>small</LabelLarge>
+            </td>
+            <td>
+                <LabelLarge>medium</LabelLarge>
+            </td>
+            <td>
+                <LabelLarge>large</LabelLarge>
+            </td>
+        </tr>
```

Comment:
**suggestion (non-blocking):** since these are headings, `th` would be a better choice than `td` here.  I think the output would look a little nicer with the headings centered.